specification/reasonable-adjustment-flag.yaml

```diff
@@ -105,6 +105,16 @@ info:
 
     For more details on sandbox testing, or to try out the sandbox using our \"Try this API\" feature, see the documentation for each endpoint.
 
+    ### Integration testing
+    Our [integration test environment](https://digital.nhs.uk/developer/guides-and-documentation/testing#integration-testing):
+    * is for formal integration testing
+    * is stateful, so persists updates
+    * includes authorisation, with options for user-restricted access (with or without smartcard) and application-restricted access
```

Comment:
These arent options for the RA INT are they?

Reply:
AFIK everything is the same as PDS we're using the same spine env. The second point i'm sure is true.
@gjugpal Can you please have a look and confirm if these points are valid?

Reply:
Points 1 and 2 are true. I'm not sure on having both user-restricted and application-restricted access for RA.

Reply:
@gjugpal Yeah we don't have application-restricted. I'll remove it. I think that's the only required change.